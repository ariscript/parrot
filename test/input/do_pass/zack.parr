def foo(n): print(bar(n)) > 1000000

and def bar(n): print(n) - 1

foo(4294967295)
