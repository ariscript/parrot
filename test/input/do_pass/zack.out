4294967295
4294967294
true
